test/e2e_node/garbage_collector_test.go

```diff
@@ -257,7 +257,10 @@ func containerGCTest(f *framework.Framework, test testRun) {
 
 // Runs containerGCTest using the docker runtime.
 func dockerContainerGCTest(f *framework.Framework, test testRun) {
-	runtime := docker.ConnectToDockerOrDie(defaultDockerEndpoint, defaultRuntimeRequestTimeoutDuration)
+	var runtime docker.DockerInterface
+	BeforeEach(func() {
+		runtime = docker.ConnectToDockerOrDie(defaultDockerEndpoint, defaultRuntimeRequestTimeoutDuration)
```

Comment:
@timstclair Yeah, good idea.

Before that, I need to figure out how ginkgo works, why our other tests don't flake, or they should just haven't flaked yet.